examples/configuration/echo.lua

```diff
@@ -0,0 +1,8 @@
+local policy_chain = require('apicast.policy_chain').default()
```

Comment:
`examples/configuration`contains many examples for different things. I think it'd be good from now on to explain what each one of them is about. A sentence at the begginning of the file should be enough.

Reply:
Good point. This one probably should be in own folder.